lib/purchaser_info_wrapper.dart

```diff
@@ -19,6 +20,9 @@ class PurchaserInfo {
   /// Set of purchased skus, active and inactive
   final List<String> allPurchasedProductIdentifiers;
 
+  /// Set of purchased skus, active and inactive
```

Comment:
I believe this docs are copy pasted from `allPurchasedProductIdentifiers`

Reply:
lol good catch, fixing